csrc/kernels/internode_ll.cu

```diff
@@ -909,14 +973,13 @@ __global__ __launch_bounds__(1024, 1) void combine(void* combined_x,
                 // Issue RDMA
                 // NOTES: for zero-copy mode, we assume the data is already in the send buffer
                 if (dst_p2p_ptr == 0)
-                    nvshmemi_ibgda_put_nbi_warp(dst_ptr, buf_ptr, num_send_bytes, dst_rank, local_expert_idx, lane_id, token_idx - offset);
+                    nvshmemi_ibgda_put_nbi_warp(dst_ptr, buf_ptr, num_send_bytes, dst_rank, local_expert_idx, lane_id, token_idx);
```

Comment:
Why should we change the variable `message_idx`?

Reply:
In overlap mode, reusing `token_idx - offset` would result in a negative `message_idx`. Here, `token_idx` represents the `id` of the token within its corresponding local_expert, which is compatible with both overlap and non-overlap modes.